Controller/ConfigController.php

```diff
@@ -141,6 +151,18 @@ public function editAction(Request $request, $id)
             throw $this->createNotFoundException('Not Found', $e);
         }
 
+        if ($request->getMethod() !== "PUT") {
+            $eventDispatcher = $this->get('event_dispatcher');
+            $event = new ConnectorCalledEvent($data->getAdapter(), $data->getName(), $data->getOptions(), __FUNCTION__);
+            $eventDispatcher->dispatch(ConnectorCalledEvent::CONNECTOR, $event);
+
+            if (!empty($event->getResponse()) && filter_var($event->getResponse(), FILTER_VALIDATE_URL) === false) {
```

Comment:
I get the idea that the getResponse method can somehow return a url as a string. Which creates this weird logic, so how about just returning a response like the name suggest.